### PR TITLE
[DOCS] Add TCA path information

### PR DESCRIPTION
**Pull request by linawolf**
Add information about where exactly the configuration is located in the TCA array

releases: main, 11.5

Fixes: #404

### CODE REVIEW

File: Documentation/ColumnsConfig/CommonProperties/FieldControl.rst [hunk 1]
```diff
@@ -5,6 +5,7 @@
 fieldControl
 ============
 
+   :Path: $GLOBALS['TCA'][$table]['columns'][$field]['config']
```
Comment (nhovratov) [inline Documentation/ColumnsConfig/CommonProperties/FieldControl.rst:8]: Should be inside `confval`

File: Documentation/ColumnsConfig/Type/Group/Properties/FieldControl.rst [hunk 2]
```diff
@@ -13,6 +13,7 @@ The field of type group can enable all common :ref:`field control options
 
 .. confval:: elementBrowser
 
+   :Path: $GLOBALS['TCA'][$table]['columns'][$field]['config']
```
Comment (nhovratov) [inline Documentation/ColumnsConfig/Type/Group/Properties/FieldControl.rst:16]: Add it for insertClipboard too?

**Review by nhovratov — CHANGES_REQUESTED** [review]
Looks good, this gives a nice breadcrumb like orientation.
Found only one path outside the confal directive and another one, which can be added.
Besideds that +1 from me.

**Comment by nhovratov** [comment]
Also, a "Fixes" can be added to the commit message for #404:

`Fixes: #404`

**Review by marble — APPROVED** [review]
I totally apreciate having that Path information. I'm not a hundred percent sure though the name will just stay "Path" if some built in functionatility is to be linked to that option. But this can surely be easily adapted to by some editorial measures.

**Pull request merged**